db/migrate/20171030200258_add_office.rb

```diff
@@ -0,0 +1,7 @@
+class AddOffice < ActiveRecord::Migration[5.1]
+  def change
+    create_table :offices do |t|
+      t.string :office_name
```

Comment:
This should just be `name`. It should also be `null: false`. Also, please add `t.timestamps`